### ISSUE TITLE
openvim does not remove openflow rules when deleting networks

### ISSUE THREAD

**Issue by alf-tierno** [nfvlabs/openmano#25]
Some unknown circunstances makes that openvim leave undeleted some openflow rules in the controller, though it deletes from database. This causes undesirable connections over the new deployment. The work arount is to clear all the rules and insert them again with "openvim openflow-clear-all" and "openvim openflow-net-reinstall"

**Comment by alogracia** [comment]
Wouldn't this workaround potentially break other scenarios running in the
VIM?

**Comment by alf-tierno** [comment]
Yes, it is very harmfull. The old rules prevails over the new ones (same priority but inserted before) spoling the new instance-scenario. Unfortunately the logs where losed and I cannot determine the reasons. I suspect that the limit of 50 in the openflow task queue can be the problem when a large scenario is deployed/deleted. Enlarged to 2000 in the git branch openflow. To be tested and waiting to see the logs when it happens again before closing this issue

**Comment by alf-tierno** [comment]
@alogracia:  answering your specific question: yes this workaround makes a network break in all running scenarios, so both commands must be launch one immediate after the other
